server/request_response.go

```diff
@@ -7,35 +7,29 @@ import (
 	"github.com/flashbots/rpc-endpoint/types"
 )
 
-func (r *RpcRequest) writeHeaderStatus(statusCode int) {
-	if r.respHeaderStatusCodeWritten {
-		return
-	}
-	r.respHeaderStatusCodeWritten = true
-	(*r.respw).WriteHeader(statusCode)
-}
+// As per JSON-RPC 2.0 Specification
+const (
```

Comment:
These consts should go into `/types/types.go` to the other JsonRpc types.

Reply:
Agreed.

Reply:
updated